client_test.go

```diff
@@ -183,10 +183,11 @@ func TestCall(t *testing.T) {
 
 func TestCallURLNoRequestPath(t *testing.T) {
 	c := qt.New(t)
-	defer c.Done()
 
 	srv := newServer()
-	c.Defer(srv.Close)
+	c.Cleanup(func() {
+		srv.Close()
+	})
```

Comment:
No need for the closure, as the Close method is already correct
for passing to Cleanup.

```suggestion
	c.Cleanup(srv.Close())
```

Same for the other occurrences.

Reply:
Done